### ISSUE TITLE
Desk Reservation app cannot find availability between reservations

### ISSUE THREAD

**Issue by melissamuse** [aprildunnam/PowerApps#117]
**Describe the bug**
Hi April!  I used this template to create a vehicle check out app.  My MAJOR bug is that it cannot find availability between bookings.  For example, if a vehicle is booked from 9a-10a and also from 2p-3pm, It will not show as available when I search for a vehicle for 12-1pm.  Any advice based on the filters that you used?

**Additional context**

**Comment by Tidomann** [comment]
The filters are correct. The two columns in your reservation list "Check out from" and "Check out to" must be date and time columns WITH "Include Time" ENABLED. By default without the "Include Time" enabled, the date and time columns will only have the date patched by the program when creating a reservation causing the filters to always detect a booking conflict.

The Instructions might want to mention this is a requirement when setting up the column

**Comment by melissamuse** [comment]
Yes!  I figured this out by accident.  I changed the format of the column to show the time so I could troubleshoot and all of my bookings started and ended at 12AM. So, the issue was with how the bookings were being created, not the filter.  Once I did that, everything worked.
Thank you so much for your reply!

Melissa Muse, 
CMP
IT Project Manager
Headquarters
861 Riverland Dr, Charleston, SC 29412
Office: +1 8437628113
***@***.***|CharlestonCountyParks.com
________________________________
